admin-guide.md

```diff
@@ -270,3 +277,19 @@ will be captured. Then `"metrics_calculate"` will be called with the
 `"output_metrics"` as the first argument and the file generated in the previous
 step as the second argument. If it exits with a non-zero return code, then the
 test is allowed to pass.
+
+## Testing Víðarr Locally
```

Comment:
These instructions would probably be better placed in the vidarr-cli README file. Also these instructions aren't really for testing vidarr so much as testing a vidarr workflow